### DISCUSSION TITLE
How to display column visibility in slideover?

### DISCUSSION THREAD

**Discussion by jadersbr** [filamentphp/filament#13769] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

Currently, it is possible to display the filters in a modal, and consequently in a slideover.

Would it be possible to do the same for column visibility? 

By default, it is a dropdown, which, if there are many records, looks unattractive on the screen.

Do you have any idea?

**Comment by leandrocfe** [comment]
_column visibility_? Sorry, what does it mean?

**Reply by jadersbr** [reply]
Hello @leandrocfe,

I'm referring to the functionality of hiding or displaying columns. The field names are displayed in a dropdown, but when we have many fields, the dropdown does not look very good. There is no scroll.

<img width="210" alt="image" src="https://github.com/user-attachments/assets/d1b21864-f487-4867-a8d6-4bf4a87d8d73">


I would like to know if there is an alternative to display these fields in a slideover, similar to the filters.

<img width="177" alt="image" src="https://github.com/user-attachments/assets/3b3b7433-7646-4a0c-b1ad-330ff482980f">

**Reply by leandrocfe** [reply]
I don't think so, but you can use `$table->columnToggleFormColumns(2)` to add a grid.

**Reply by jadersbr** [reply]
Based on your tip, I found something that will work as a temporary solution.

`->columnToggleFormMaxHeight('300px')`

![image](https://github.com/user-attachments/assets/c4ca15df-56c4-4581-8eea-0bd2cbe35c98)
